winui/Cartesian-Charts/Bar.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: Bar Chart in WinUI Chart control | Syncfusion
+description: Learn here all about bar chart support in Syncfusion WinUI Chart (SfCartesianChart) control.
+platform: WinUI
+control: SfCartesianChart
+documentation: ug
+---
+
+# Bar Chart in WinUI Chart (SfCartesianChart)
+
+A bar chart uses bars to represent data points and compare values across different categories. To render bar chart, initialize the [SfCartesianChart](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SfCartesianChart.html) and switch the chart X and Y axes by using the [IsTransposed](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html#Syncfusion_Maui_Charts_SfCartesianChart_IsTransposedProperty) property as true. Then, create the column chart. To render a column chart, create an instance of [ColumnSeries](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ColumnSeries.html), and add it to the [Series](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SfCartesianChart.html#Syncfusion_UI_Xaml_Charts_SfCartesianChart_Series) collection property of [SfCartesianChart](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SfCartesianChart.html).
+
```

Comment:
@Harsha-SF4223 , change the bar chart description based on suggestion

Reply:
I have changed the description.